src/test/java/de/tonypsilon/bmm/backend/club/service/ClubServiceTest.java

```diff
@@ -49,7 +50,7 @@ void setUp() {
     void testGetAllClubs() {
         when(clubRepository.findAll()).thenReturn(List.of(club1, club2, club3));
         Collection<ClubData> actual = clubService.getAllClubs();
-        assertEquals(3, actual.size());
+        assertThat(actual.size()).isEqualTo(3);
```

Comment:
@ujwalkumar1995 in cases like this, you can use assertThat(actual).hasSize(3). These very readable methods is the power of AssertJ and the reasoning for changing the assertEquals method to AssertJ assertions :)

Reply:
Thanks for the feedback. Will make these changes and keep in mind.